driver-core/src/main/java/com/datastax/driver/core/TypeCodec.java

```diff
@@ -159,7 +159,7 @@ static TupleCodec tupleOf(TupleType type) {
     }
 
     /* This is ugly, but not sure how we can do much better/faster
-     * Returns null if it's doesn't correspond to a known type.
+     * Throws IllegalArgumentException if it's doesn't correspond to a known type.
```

Comment:
Would be good to use `@throws` annotation here since this is an unchecked exception.